### ISSUE TITLE
Any plans to add Master Server Query Protocol in the near future?

### ISSUE THREAD

**Issue by Runninginsilence1** [cemahseri/SteamQuery.NET#6]

**Comment by cemahseri** [comment]
Yes, I do plan to add support for both Master Server Query Protocol, Steam Web API (for querying a master server in the first place), and the Source RCON Protocol in this order. I cannot give an estimated time but I would like to add it as soon as possible.

**Comment by cemahseri** [comment]
Implemented in 49a116eb6fc66696307065c053bd4cd0c2eddf5b.

**Comment by Runninginsilence1** [comment]
Cooooooooooooool

**Comment by cemahseri** [comment]
you;
![Image](https://github.com/user-attachments/assets/575e9cb4-b5ee-49cc-80a4-1219d2056d30)
